.vscode/settings.json

```diff
@@ -13,5 +13,6 @@
   },
   "[json]": {
     "editor.defaultFormatter": "esbenp.prettier-vscode"
-  }
+  },
+  "python.pythonPath": "C:\\Python27\\ArcGIS10.7\\python.exe"
```

Comment:
I dont think this is needed in this repo

Reply:
yep, not sure how it got there tbh.